frontend/packages/knative-plugin/src/utils/fetch-dynamic-eventsources-utils.ts

```diff
@@ -43,6 +43,7 @@ export const fetchEventSourcesCrd = async () => {
             names: { kind, plural, singular },
           },
         } = crd;
+        const label = kind.replace(/([A-Z])/g, ' $1').trim();
```

Comment:
We have been setting wrong values to the label ex: `apiserversource` whereas it should be  `Api Server Source` because this label can be used to show title on pages like the list page and `apiserversource` should be assigned to `singular` prop.
Since we don't receive label exactly from the response this regex convert kind into a label
```
ApiServerSource (kind) => Api Server Source (label)
```
Similarly
```
ApiServerSource (kind) => Api Server Sources (labelPlural)
```